packages/composer/lib/proxy.js

```diff
@@ -213,6 +214,10 @@ module.exports = fp(async function (app, opts) {
             headers: res.headers
           })
           reply.send(res.stream)
+        },
+        onError: (reply, { error }) => {
+          app.log.error({ error: ensureLoggableError(error) }, 'Error while proxying request to another service')
```

Comment:
I may miss some context here, shouldn't be done in the logger serializers?

Reply:
You are right in theory. Lol.
The composer, as any other service, executes in its own thread. All the output is intercepted by the main thread that in place sends them to the configured serializers.